crates/egui/src/containers/scroll_area.rs

```diff
@@ -499,6 +499,11 @@ struct Prepared {
 
     scrolling_enabled: bool,
     stick_to_end: Vec2b,
+
+    /// If there was a scroll target before the ScrollArea was added this frame, it's
+    /// not for us to handle so we save it and restore it after this ScrollArea is done.
+    saved_scroll_target: [Option<pass_state::ScrollTarget>; 2],
```

Comment:
There is also `FrameState::scroll_delta`, I think we should do the same with that?

Reply:
@lucasmerlin There's no `FrameState` anymore so I think this comment is not valid anymore?